Traceback (most recent call last):
  File "/home/pyhome/server/speacker/speacker_demon.py", line 89, in <module>
    % (2, 16, 16000)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 99-104: ordinal not in range(128)
